nsv13/code/game/gamemodes/pvp/items.dm

```diff
@@ -431,3 +431,21 @@
 	network = list("syndicate")
 
 /obj/machinery/camera/syndicate/autoname
+	var/number = 0
```

Comment:
why not just make the default value 1?

Reply:
Changing that value to one causes areas with multiple cameras to be named area 2, area 3, area 4. skipping 1.

Reply:
Ah I didn't see the part where it did `number +1` in the code (which is honestly pretty fucked up but alright)